modules/standard/Subprocess.chpl

```diff
@@ -1039,13 +1056,32 @@ module Subprocess {
     on home {
       err = qio_send_signal(pid, signal:c_int);
     }
-    if err then try ioerror(err, "in subprocess.sendPosixSignal, with signal " + signal:string);
+    if (err == EINVAL) {
+      throw new IllegalArgumentError("Signal " + signal: string +
+                                     " is not a valid POSIX signal");
```

Comment:
Should we be wrapping EINVAL with IllegalArgumentError all the time? I don't mean to slow down this PR with this question.

Reply:
Opened #25509